src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+use Symfony\Component\Cache\Adapter\AdapterInterface;
+use Symfony\Component\Cache\Adapter\ApcuAdapter;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+use Symfony\Component\Cache\Adapter\DoctrineAdapter;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+use Symfony\Component\Cache\Adapter\MemcachedAdapter;
+use Symfony\Component\Cache\Adapter\PdoAdapter;
+use Symfony\Component\Cache\Adapter\ProxyAdapter;
+use Symfony\Component\Cache\Adapter\RedisAdapter;
+use Symfony\Component\Cache\Adapter\TagAwareAdapter;
+use Symfony\Component\Cache\Marshaller\DefaultMarshaller;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+use Symfony\Contracts\Cache\CacheInterface;
+use Symfony\Contracts\Cache\TagAwareCacheInterface;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('cache.app')
+            ->parent('cache.adapter.filesystem')
+            ->public()
+            ->tag('cache.pool', ['clearer' => 'cache.app_clearer'])
+
+        ->set('cache.app.taggable', TagAwareAdapter::class)
+            ->args([service('cache.app')])
+
+        ->set('cache.system')
+            ->parent('cache.adapter.system')
+            ->public()
+            ->tag('cache.pool')
+
+        ->set('cache.validator')
+            ->parent('cache.system')
+            ->private()
+            ->tag('cache.pool')
+
+        ->set('cache.serializer')
+            ->parent('cache.system')
+            ->private()
+            ->tag('cache.pool')
+
+        ->set('cache.annotations')
+            ->parent('cache.system')
+            ->private()
+            ->tag('cache.pool')
+
+        ->set('cache.property_info')
+            ->parent('cache.system')
+            ->private()
+            ->tag('cache.pool')
+
+        ->set('cache.messenger.restart_workers_signal')
+            ->parent('cache.app')
+            ->private()
+            ->tag('cache.pool')
+
+        ->set('cache.adapter.system', AdapterInterface::class)
+            ->abstract()
+            ->factory([AbstractAdapter::class, 'createSystemCache'])
+            ->tag('cache.pool', ['clearer' => 'cache.system_clearer', 'reset' => 'reset'])
```

Comment:
I know that in the original XML the tags were before the args, so you did this perfectly well. But it looks like in the PHP syntax we want to display args first and then the tags, so could you please reorder these lines? Thanks!

Reply:
done